### ISSUE TITLE
GUM_vlog_pregnant: trinary RST branch in lisp binary

### ISSUE THREAD

**Issue by lgessler** [amir-zeldes/gum#80]
If I'm reading [lisp_binary/GUM_vlog_pregnant.dis](https://github.com/amir-zeldes/gum/blob/master/rst/lisp_binary/GUM_vlog_pregnant.dis#L298L305) right, there's a trinary branch here:

```
                      ( Nucleus (span 102 103) (rel2par joint)
                        ( Nucleus (leaf 102) (rel2par span) (text _!and you just feel awful_!) )
                        ( Satellite (leaf 103) (rel2par circumstance) (text _!thinking ._!) )
                        ( Satellite (span 104 107) (rel2par elaboration)
                          ( Nucleus (span 104 106) (rel2par span)
                            ( Nucleus (span 104 105) (rel2par same-unit)
                              ( Nucleus (leaf 104) (rel2par span) (text _!The thought_!) )
                              ( Satellite (leaf 105) (rel2par elaboration) (text _!of telling someone_!) )
                            )
                            ( Nucleus (leaf 106) (rel2par same-unit) (text _!makes me want to burst into tears ._!) )
                          )
                          ( Satellite (leaf 107) (rel2par elaboration) (text _!All these things going onto my head ._!) )
                        )
                      )
```

Maybe there's something I'm misunderstanding? Janet and I came across this doing something unrelated

**Comment by amir-zeldes** [comment]
Thanks for reporting, this was the result of an incorrect hierarchy warning that wasn't caught in the annotation interface and messed up the conversion script. We should really build a validation to catch these in the build bot in case we miss them during annotation...

**Comment by lgessler** [comment]
I sketched some code referenced in #76 for validating build bot output in general. We can chat about it sometime--I tried to set up some structure to make writing validations easy. See e.g. [this rule for ensuring only the sanctioned items appear with the deprel `fixed`](https://github.com/lgessler/gum/commit/d120df10dd8491dcde293ea07fa70c4d9270ff9c#diff-5103169fd5fa8abca11bee8852e79306292501c5def71c93b92c8d097a84da2fR218).
